### DISCUSSION TITLE
4.9.8?

### DISCUSSION THREAD

**Discussion by marmarama** [super-linter/super-linter#3476] [General]
Hi,

super-linter 4.9.7 has a bug (#3406) that prevents tflint from working properly. This was fixed in #3414 but has not been released yet.

I'm currently working to replace all our linting actions with super-linter but the tflint bug in 4.9.7 is a blocker. 

Is a 4.9.8 release possible sometime in the near future? I know I can build my own fork but that comes with a substantial amount of overhead in the environment I'm working in.

Thank you for your help and your work on super-linter, it's super-useful!

@lindluni

**Comment by lindluni** [comment]
Hey @marmarama 

I will get this done today!

**Reply by marmarama** [reply]
Amazing, thank you!

**Comment by marmarama** [comment]
Hey @lindluni

Just a quick nudge on this, sorry to be a pain!

**Comment by YElyousfi** [comment]
Hey @lindluni 

Before 4.9.8 is released, can we assess PR #3540 to bump actionlint to the latest version? There are bugs in the version of actionlint being used in 4.9.7 that are a blocker for me as well as I try to centralize our linting actions similar to @marmarama

**Comment by lindluni** [comment]
4.10.0 is released, was a long game waiting on our fixes to ts-standard to get merged and then update months worth of dependency updates in the backlog behind it.

**Reply by marmarama** [reply]
Amazing, thank you!
